test/root.test.js

```diff
@@ -1,14 +1,10 @@
 import React from 'react';
-import { render } from '@testing-library/react';
-import { Provider } from 'react-redux';
-import store from '../src/redux/store';
+import { render } from './jest.setup';
 
 import App from '../src/components/app';
 
 test('on initial render, display something', () => {
   render(
-    <Provider store={store}>
       <App />
-    </Provider>
   );
 });
```

Comment:
fix this

Reply:
I have create a render method that wraps all components in redux store so this provider is no longer needed as this custom render already have the store.